### PR TITLE
Error message was not direct enough.

### PR DESCRIPTION
**Pull request by voronoipotato**
Scott Nimrod, otherwise completely capable F# developer was flummoxed by this in a live stream. Added clear resolution steps to error message. It's a rather roundabout way to say "You've got an extra space there".

https://youtu.be/_6jV1n0fL00?t=10m25s

### CODE REVIEW

File: src/fsharp/FSComp.txt [hunk 1]
```diff
@@ -1041,8 +1041,8 @@ lexHashBangMustBeFirstInFile,"#! may only appear as the first line at the start
 1186,parsInvalidProperty,"Invalid property getter or setter"
 1187,parsIndexerPropertyRequiresAtLeastOneArgument,"An indexer property must be given at least one argument"
 1188,tastInvalidAddressOfMutableAcrossAssemblyBoundary,"This operation accesses a mutable top-level value defined in another assembly in an unsupported way. The value cannot be accessed through its address. Consider copying the expression to a mutable local, e.g. 'let mutable x = ...', and if necessary assigning the value back after the completion of the operation"
-1189,parsNonAdjacentTypars,"Type parameters must be placed directly adjacent to the type name, e.g. \"type C<'T>\", not     type \"C   <'T>\""
-1190,parsNonAdjacentTyargs,"Type arguments must be placed directly adjacent to the type name, e.g. \"C<'T>\", not \"C  <'T>\""
+1189,parsNonAdjacentTypars,"Remove spaces after type, e.g. \"type C<'T>\", not type \"C   <'T>\" type parameters must be placed directly adjacent to the type name"
```
Comment (cartermp) [inline src/fsharp/FSComp.txt]: Suggested change:

```
Remove spaces between the type name and type parameter, e.g. \"type C<'T>\", not type \"C   <'T>\". Type parameters must be placed directly adjacent to the type name
```

This way it's a bit more specific about what the issue is. It also fixes a run-on sentence.

Comment (KevinRansom) [reply src/fsharp/FSComp.txt]: Did you mean to delete this line without replacing it?

-1190,parsNonAdjacentTyargs,"Type arguments must be placed directly adjacent to the type name, e.g. \"C<'T>\", not \"C  <'T>\""

Comment (KevinRansom) [reply src/fsharp/FSComp.txt]: Like a twerp I looked at the conversation, not the files :-(

Comment (voronoipotato) [reply src/fsharp/FSComp.txt]: @KevinRansom what does that mean? Is something wrong?

**Comment by KevinRansom** [comment]
@dotnet-bot test this please

File: src/fsharp/FSComp.txt (hunk 1 above)
Comment (KevinRansom) [reply src/fsharp/FSComp.txt]: @voronoipotato no, I nothing is wrong.

**Comment by cartermp** [comment]
@dotnet-bot test this please

@voronoipotato would you like to make the suggested change?

**Comment by voronoipotato** [comment]
sure I can do that.

**Comment by voronoipotato** [comment]
sorry about the bumpy ride here, first time doing a pull request on a project with a significant number of contributors.

**Comment by cartermp** [comment]
No worries!

**Comment by cartermp** [comment]
@brettfo Any idea about what this error in CI is about?

> D:\j\w\release_ci_pa---3f142ccc\Packages\XliffTasks.0.2.0-beta-000076\build\XliffTasks.targets(90,5): error : '..\xlf\FSComp.txt.en.xlf' is out-of-date with 'obj\net40\FSComp.resx'. Run `msbuild /t:UpdateXlf` to update .xlf files or set UpdateXlfOnBuild=true to update them on every build, but note that it is strongly discouraged to set UpdateXlfOnBuild=true in official/CI build environments as they should not modify source code during the build. [D:\j\w\release_ci_pa---3f142ccc\src\fsharp\FSharp.Compiler.Private\FSharp.Compiler.Private.fsproj]

**Comment by cartermp** [comment]
@dotnet-bot test this please

**Comment by KevinRansom** [comment]
@cartermp phillip ... that is a notification that the xlif files are not uptodate.  You need to ensure they are upto date.  The cli machine does not edit PRs

use these commands to update the pr:
````
cd $(yourroot)\src\fsharp\FSharp.Compiler.Private
msbuild /t:UpdateXlf
````

**Comment by voronoipotato** [comment]
Oh beans I deleted my repository. Is this a problem?

**Comment by KevinRansom** [comment]
@voronoipotato no probs mate, apparently git hub is smart enough to cope with that kind of thing.

Kevin

**Pull request merged**